samples/ipc/ipc_service/README.rst

```diff
@@ -13,7 +13,14 @@ Overview
 ********
 
 The sample application tests throughput of the IPC service with available backends.
-Currently, the sample supports OpenAMP RPMSG and ICMSG backends.
+Currently, the sample supports the following backends:
+
+* the :ref:`zephyr:nrf5340dk_nrf5340` board
+  * the `OpenAMP`_ library
+  * :ref:`zephyr:ipc_service_backend_icmsg`
+
+* the :ref:`zephyr:nrf54h20dk_nrf54h20` board:
+  * :ref:`zephyr:ipc_service_backend_icbmsg`
```

Comment:
```suggestion
Currently, the sample supports the following backends:

* the :ref:`zephyr:nrf5340dk_nrf5340` board

  * the `OpenAMP`_ library
  
  * :ref:`zephyr:ipc_service_backend_icmsg`

* the :ref:`zephyr:nrf54h20dk_nrf54h20` board:

  * :ref:`zephyr:ipc_service_backend_icbmsg`
```
It seems to me that there should be blank line between the bullets.

Reply:
Same level bullets do not need a blank line in between.

Reply:
Done